bitrise.yml

```diff
@@ -1495,30 +1495,14 @@ workflows:
                 -destination "generic/platform=iOS" \
                 -derivedDataPath "/Users/vagrant/git/DerivedData"
     - deploy-to-bitrise-io@2: {}
-    - script@1.1:
-        is_always_run: true
-        title: Verify App signature, Test, and Runner
-        inputs:
-        - content: |
-            echo "see files created"
-            ls /Users/vagrant/git/DerivedData
-            ls /Users/vagrant/git/DerivedData/Build/Products
-            set -euxo pipefail
-            echo "-- verify test and runner signature --"
-            codesign --verify --deep --strict --verbose DerivedData/Build/Products/Debug-iphoneos/Client.app
-            echo "-- test app --"
-            xcodebuild -project /Users/vagrant/git/Client.xcodeproj \
-                -scheme Fennec_Enterprise \
-                -sdk iphoneos \
-                -derivedDataPath "/Users/vagrant/git/DerivedData" test-without-building | xcpretty | tee xcodebuild_fennec.log
     - script@1.1:
         title: Create .zip artifact for Firebase Testing
         inputs:
         - content: |
             set -euxo pipefail
             echo "-- create .zip artifact --"
             cd DerivedData/Build/Products
-            zip -r Fennec_Enterprise.zip Debug-iphones Fennec_Enterprise_PerformanceTestPlan_iphoneos16.4-arm64.xctestrun
+            zip -r Fennec_Enterprise.zip Debug-iphones Fennec_Enterprise_PerformanceTestPlan_iphoneos16.4-x86_64.xctestrun
```

Comment:
This path is not correct... There is not any Debug-iphones folder created... I think we need to zip:
- Fennec_Enterprise-iphoneos
And
- Fennec_Enterprise_PerformanceTestPlan_iphoneos16.4-arm64.xctestrun